src/components/Cart.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react'
+import { useNavigate } from 'react-router-dom';
+import { orderNow } from "../config/firebase";
+import { RxCross2 } from "react-icons/rx";
+import Swal from 'sweetalert2'
+
+function Cart() {
+    const navigate = useNavigate();
+    const cartItemsCount = 2;
+    const cartItemsList = ['Chicken burger', 'Pizza'];
+    const totalPrice = 24;
+    const userDetails = {
```

Comment:
Hard-coded userDetails and resDetails would be removed once MenuDetails is completed?

Reply:
Yes, it will be removed once the login and restaurants page changes are completed.